cpp/modules/luma.gl/core/src/model.cpp

```diff
@@ -39,6 +41,7 @@ Model::Model(wgpu::Device device, const Model::Options& options) {
   descriptor.vertexStage.module = this->vsModule;
   descriptor.cFragmentStage.module = this->fsModule;
   descriptor.cColorStates[0].format = options.textureFormat;
+  descriptor.primitiveTopology = wgpu::PrimitiveTopology::TriangleStrip;
```

Comment:
We need to be able to specify this in the model options, different layers use different primitive topologies.